MigrationExecutorApp/ChangeFeedProcessortHost.cs

```diff
@@ -1,38 +1,55 @@
 ﻿namespace MigrationConsoleApp
 {
     using System;
+    using System.Collections.Concurrent;
+    using System.Collections.Generic;
     using System.Diagnostics;
+    using System.IO;
+    using System.Linq;
+    using System.Runtime.Serialization.Json;
+    using System.Text;
+    using System.Threading;
     using System.Threading.Tasks;
+    using System.Xml.Linq;
+    using System.Xml.XPath;
     using Azure.Storage.Blobs;
+    using Microsoft.Azure.Cosmos;
     using Microsoft.Azure.Documents;
-    using Microsoft.Azure.Documents.ChangeFeedProcessor;
-    using Microsoft.Azure.Documents.ChangeFeedProcessor.PartitionManagement;
-    using Microsoft.Azure.Documents.Client;
+    using Newtonsoft.Json;
+
 
     public class ChangeFeedProcessorHost
     {
         private MigrationConfig config;
-        private IChangeFeedProcessor changeFeedProcessor;
-        private DocumentClient destinationCollectionClient;
+        private ChangeFeedProcessor changeFeedProcessor;
+        private static CosmosClient destinationCollectionClient;
+        private static CosmosClient sourceCollectionClient;
+        private static Container containerToStoreDocuments;
+        private readonly string SourcePartitionKeys;
+        private readonly string TargetPartitionKey;
+        BlobContainerClient containerClient = null;
+
+
+        public ChangeFeedProcessorHost(MigrationConfig config)
+        {
+            this.config = config;
+            SourcePartitionKeys = config.SourcePartitionKeys;
+            TargetPartitionKey = config.TargetPartitionKey;
+            destinationCollectionClient = new CosmosClient(config.DestUri, config.DestSecretKey, new CosmosClientOptions() { AllowBulkExecution = true });
+            sourceCollectionClient = new CosmosClient(config.MonitoredUri, config.MonitoredSecretKey, new CosmosClientOptions() { AllowBulkExecution = true });
```

Comment:
Source collection does not need Bulk on, Bulk should be set for the destination collection, because that's where we write?

Reply:
Thanks, will resolve